includes/REST_API/Stories_Controller.php

```diff
@@ -93,6 +105,178 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
 	}
 
+	/**
+	 * Pre-fills story with demo content.
+	 *
+	 * @return string Pre-filled post content if applicable, or the default content otherwise.
+	 */
+	private function get_demo_content() {
```

Comment:
This need a unit test.

Reply:
I can add a test verifying that this returns some JSON content, but that's about it. Not very helpful.

Reply:
- What about if the file isn't readable?
- What if the file is invalid invalid json?
- What if one or all of the replacement below doesn't work?

Unit tests that cover all these case seem like valid unit tests to me.

Reply:
The file in question is always readable and valid JSON, so testing that would be rather pointless.

The only valid test I could see here is checking that the final output is still valid JSON and that there are no placeholders left.

I'll take a pass at this now.

Reply:
This method could live in the Story type class if you wish, using the `web_stories_default_content` filter

Reply:
Good point. I've removed that filter I had originally to keep things simple and because I didn't want anyone else to interfere with this. We'd need to be in full control of the content.

`Story_Post_Type` class doesn't feel super fitting either though. Technically I could move this into its own class.

Reply:
It's own class might make this easier to test.